src/components/CardHorizontal.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC } from "react";
+import styled, { useTheme } from "styled-components";
+
+import { Icon, IconName } from "./Icon";
+import { Text } from "./Text";
+import { Title } from "./Title";
+
+const CardContent = styled.div`
+  padding: 45px 25px;
+  flex: 1 1 auto;
+`;
+
+const CardTitle = styled(Title)`
+  margin-bottom: 29px;
+`;
+
+const CardIcon = styled(Icon)`
+  margin-bottom: 36px;
+`;
+
+const CardActions = styled.div`
+  margin-top: 30px;
+`;
+
+const CardImage = styled.div`
+  flex: 0 0 300px;
+  overflow: hidden;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.sm}) {
+    display: none;
+  }
+`;
+
+const CardImageInner = styled.div`
+  transform: scale(1);
+  transition: transform 0.5s ease-out;
+  width: 100%;
+  height: 100%;
+`;
+
+const CardContainer = styled.div`
+  width: 100%;
+  border-top-right-radius: ${({ theme }) => theme.card.borderRadius};
+  border-bottom-left-radius: ${({ theme }) => theme.card.borderRadius};
+  overflow: hidden;
+  background-color: ${({ theme }) => theme.color.lightGray};
+  display: inline-flex;
+  transition: all 0.5s ease-out;
+
+  &:hover {
+    background-color: ${(props) => props.theme.color.white};
+    box-shadow: ${(props) => props.theme.card.boxShadow};
+
+    ${CardImageInner} {
+      transform: scale(1.25);
+    }
+  }
+`;
+
+type CardHorizontalProps = {
+  title: string;
+  icon?: IconName;
+  description?: string | JSX.Element;
+  actions?: JSX.Element;
+  image?: JSX.Element;
```

Comment:
C'è qualche differenza dall'utilizzo di JSX.Element e React.ReactElement?

Reply:
La risposta è qui: https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement
Dipende probabilmente anche che cosa vogliamo passare. Per come lo usiamo dovrebbe cambiar poco, se ce ne fosse necessità lo cambieremo.